panamaconverter/src/PanamaCIRCTConverter.scala

```diff
@@ -1658,6 +1658,14 @@ private[panamaconverter] object PanamaCIRCTConverter {
     cvt
   }
 
```

Comment:
Yeah… this todo… I think we can do it in the following pr

Reply:
The test references the object, should be refactored in follow-up PRs.